src/main/resources/data-mapping.json

```diff
@@ -718,6 +718,624 @@
+        },
+        {
+          "field": "APPROVAL_STATUS",
+          "dataSource": {
+            "default": "APPROVED",
+            "translation": "lookupWorkflowStatusId",
+            "translateDefault": true
+          }
+        },
+        {
+          "field": "VENDOR",
+          "dataSource": {
+            "default": "GOBI Library Solutions"
+          }
+        },
+        {
+          "field": "FUND_PERCENTAGE",
+          "dataSource": {
+            "default": "100"
+          }
+        },
+        {
+          "field": "PRODUCT_ID_TYPE",
+          "dataSource": {
+            "default": "ISBN"
+          }
+        },
+        {
+          "field": "SUBSCRIPTION_FROM_DATE",
+          "dataSource": {
+            "default": "NA"
+          }
+        },
+        {
+          "field": "SUBSCRIPTION_INTERVAL",
+          "dataSource": {
+            "default": "NA"
+          }
+        },
+        {
+          "field": "SUBSCRIPTION_TO_DATE",
+          "dataSource": {
+            "default": "NA"
+          }
+        },
+        {
+          "field": "RECEIPT_DUE",
+          "dataSource": {
+            "default": true
+          }
+        },
+        {
+          "field": "ACQUISITION_METHOD",
+          "dataSource": {
+            "default": "Purchase at Vendor System"
+          }
+        },
+        {
+          "field": "CLAIM",
+          "dataSource": {
+            "default": "true"
+          }
+        },
+        {
+          "field": "COLLECTION",
+          "dataSource": {
+            "default": false
+          }
+        },
+        {
+          "field": "PO_LINE_WORKFLOW_STATUS",
+          "dataSource": {
+            "default": "Order Sent"
+          }
+        },
+        {
+          "field": "PO_LINE_RECEIPT_STATUS",
+          "dataSource": {
+            "default": "Awaiting Receipt"
+          }
+        },
+        {
+          "field": "PO_LINE_PAYMENT_STATUS",
+          "dataSource": {
+            "default": "Awaiting Payment"
+          }
+        },
+        {
+          "field": "SOURCE_ID",
+          "dataSource": {
+            "default": "Loaded from Vendor (API)"
+          }
+        },
+        {
+          "field": "MANUAL_RENEWAL",
+          "dataSource": {
+            "default": "NA"
+          }
+        },
+        {
+          "field": "RENEWAL_CYCLE",
+          "dataSource": {
+            "default": "NA"
+          }
+        },
+        {
+          "field": "RENEWAL_DATE",
+          "dataSource": {
+            "default": "NA"
+          }
+        },
+        {
+          "field": "RENEWAL_REVIEW_PERIOD",
+          "dataSource": {
+            "default": "NA"
+          }
+        },
+        {
+          "field": "RENEWAL_INTERVAL",
+          "dataSource": {
+            "default": "NA"
+          }
+        },
+        {
+          "field": "VENDOR_REF_NO_TYPE",
+          "dataSource": {
+            "default": "Supplier's unique order line reference number"
```

Comment:
I think that was a note to the developer, not something to used literally...

Reply:
If that's the case then we'll need to update previous mappings, too.   Do you know what VENDOR_REF_NO_TYPE's default value could be? Neither the spreadsheet nor the gobi order xsd has more description.

Reply:
I think the field **source_id** is also similar which is mentioned as **Loaded from Vendor**, i think that needs to be changed as well

Reply:
The `source_id` is likely a UUID to an entry in the `source` table. However, what to actually map it to is unclear. It is also unclear how `source` is to be used, it is like many other tables, just `code`, `description`, `id` and `po_line_id` and the examples all have `null` for the `code` and an empty string for the `description`. There is no reference data in mod-orders-storage as well.

I referred to these 2 mappings in #22 as completely bogus. I am in favor of removing them and creating a US to research and add the appropriate mapping value.